api/cpp/include/slint.h

```diff
@@ -589,6 +588,14 @@ class Model
         model_row_data_dirty_property.get();
     }
 
+    /// \private
+    /// Convenience function that calls `track_row_data_changes` before returning `row_data`
+    std::optional<ModelData> row_data_tracked(int row)
```

Comment:
`row_data` is const, so I think this function should be const, too :)

```suggestion
    std::optional<ModelData> row_data_tracked(int row) const
```

Reply:
hehe...  I wanted to do that, but then i need to const_cast to call track_row_data_changes.
I also considered making track_row_data_changes `const` as well, but then we need to make tracked_rows `mutable`

So i thought i'd leave it non-const and maybe we put back the const later.  What do you think?  should track_row_data_changes be const?

Reply:
Yeah I think it should be const (along with a const cast inside). The inconsistency is worse IMO :)

Reply:
> Yeah I think it should be const

Do you mean row_data_untracked or track_row_data_changes?

Reply:
I mean `row_data_untracked`, since that's public documented API.

Reply:
I didn't really made it public since the other stuff were also private.